server/util/src/canonical_json.rs

```diff
@@ -0,0 +1,77 @@
+use std::{collections::BTreeMap, fmt};
```

Comment:
is this actually needed ? https://docs.serde.rs/serde_json/enum.Value.html#variant.Object

Reply:
`By default the map is backed by a BTreeMap`